README.md

```diff
@@ -101,12 +101,12 @@ vars:
 ```
 
 ### Changing the Build Schema
-By default, this package will build the Recharge transformation models within a schema titled (<target_schema> + `recharge`) in your destination. If this is not where you would like your Recharge data written, add the following configuration to your root `dbt_project.yml` file:
+By default, this package will build the Recharge transformation models within a schema titled (<target_schema> + `recharge`) in your destination. If this is not where you would like your Recharge data written, add the following configuration to your root `dbt_project.yml` file: 
 
 ```yml
-models:
-    recharge:
-      +schema: my_new_schema_name # leave blank for just the target_schema
+models: 
+    recharge: 
+      +schema: my_new_schema_name # leave blank for just the target_schema 
```

Comment:
Don't forget to also include the source schema details here as well since it is installed by default.